Source/WebKit/Shared/WebEvent.serialization.in

```diff
@@ -107,6 +110,40 @@ class WebKit::WebTouchEvent : WebKit::WebEvent {
     bool canPreventNativeGestures();
 #endif
 };
+
+[Nested] enum class WebKit::WebPlatformTouchPoint::State : uint8_t {
+    Released,
+    Pressed,
+    Moved,
+    Stationary,
+    Cancelled
+};
+
+[CustomHeader] class WebKit::WebPlatformTouchPoint {
+#if PLATFORM(IOS_FAMILY)
+    unsigned identifier()
+    WebCore::IntPoint location()
+    WebKit::WebPlatformTouchPoint::State phase()
+#endif
+#if PLATFORM(IOS_FAMILY) && ENABLE(IOS_TOUCH_EVENTS)
+    double radiusX()
+    double radiusY()
+    double rotationAngle()
+    double force()
+    double altitudeAngle()
+    double azimuthAngle()
+    WebKit::WebPlatformTouchPoint::TouchType touchType()
+#endif
+#if !PLATFORM(IOS_FAMILY)
+    uint32_t id()
+    WebKit::WebPlatformTouchPoint::State state()
+    const WebCore::IntPoint& screenPosition()
+    const WebCore::IntPoint& position()
+    const WebCore::IntSize& radius()
```

Comment:
Do these have to be const refs? I don't see precedence in other `serialization.in` files